pubspec.lock

```diff
@@ -0,0 +1,1596 @@
+# Generated by pub
```

Comment:
IMO, don't need this file

Reply:
what happen if developer upgrade the dependencies without push this file to upstream?